### ISSUE TITLE
(0.5) Missing support for generator expressions

### ISSUE THREAD

**Issue by TotalVerb** [tonyhffong/Lint.jl#141]
```julia
julia> lintstr("Dict(β => β for β in [1, 2, 3])")
2-element Array{Lint.LintMessage,1}:
 none:1 E321 β: use of undeclared symbol
 none:1 E321 β: use of undeclared symbol
```

I'll take care of most of these issues. Just recording them here to keep track.

**Comment by Michael-Klassen** [comment]
> I'll take care of most of these issues. Just recording them here to keep track.

Thank you and good job!
I'll try to review and merge your contributions in a timely manner.

**Comment by TotalVerb** [comment]
This support was actually added in https://github.com/tonyhffong/Lint.jl/pull/147. I forgot to mark this issue as closed.
